src/VerifyExamples.elm

```diff
@@ -60,11 +60,15 @@ update msg model =
 
         CompileModule ( moduleName, fileText ) ->
             let
-                generatedTestFileText =
+                generatedTests =
                     Parser.parse fileText
-                        |> Compiler.compile moduleName
+                        |> List.concatMap (Compiler.compile moduleName)
             in
-                model ! [ writeFile ( moduleName, generatedTestFileText ) ]
+            ( model
+            , generatedTests
+                |> List.map writeFile
+                |> Cmd.batch
```

Comment:
It would be better to send these out in one go, rather than batching them

Reply:
i.e `writeFiles : List (String, String) -> Cmd msg`

Reply:
oki